rewrite-java/src/test/java/org/openrewrite/java/LombokUtilityClassTest.java

```diff
@@ -0,0 +1,385 @@
+/*
+ * Copyright 2023 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java;
+
+import org.junit.jupiter.api.Nested;
+import org.junit.jupiter.api.Test;
+import org.openrewrite.test.RewriteTest;
+
+import static org.openrewrite.java.Assertions.java;
+
+/**
+ * Cases to test:
+ * - inheritance
+ * - instantiations of changed classes
+ * - constructor
+ */
+class LombokUtilityClassTest implements RewriteTest {
+
+    @Nested
+    class ShouldApplyLombokUtility implements RewriteTest {
+
+        @Test
+        void givenOneStaticMethod() {
+            rewriteRun(
+                    recipeSpec -> recipeSpec.recipe(new LombokUtilityClass()),
+                    java(
```

Comment:
Suggest to 
1. override defaults to define the recipe only once; `spec.recipe(Recipe)` calls can be removed from all other tests too
2. nested classes don't have a to also implement `RewriteTest`

```suggestion
    @Override
    public void defaults(RecipeSpec spec) {
        spec.recipe(new LombokUtilityClass());
    }

    @Nested
    class ShouldApplyLombokUtility {

        @Test
        void givenOneStaticMethod() {
            rewriteRun(
                    java(
```